src/components/StudyPane/InfoPane/Motif/Root.tsx

```diff
@@ -74,17 +74,21 @@ const Root = ({
         })
         ctxSetRootsColorMap(newMap);
     }
-  
+
     return (
-        <div>
-            <div className="flex flex-wrap pb-8">
-                {
-                    rootWords.map((root, index) => (
+        <div className="flex-col h-full">
+            <div
+                style={{ height: '70%' }}
```

Comment:
With the new y-scroll bar for rootblock, I am not sure if we still need this styling.

with height: '70%'
![image](https://github.com/user-attachments/assets/4dec1ae6-c301-4725-894f-ed112a444931)
![image](https://github.com/user-attachments/assets/f98813d9-9939-41bf-8e6a-fa3a1d0348ba)

without: 
![image](https://github.com/user-attachments/assets/40832e32-58ac-4184-b5ca-a9ae52bdadc9)
![image](https://github.com/user-attachments/assets/3fb4b7de-e05d-4247-9412-80e8e03b43a9)
![image](https://github.com/user-attachments/assets/1ad122dc-ba67-49f0-8aa4-f1cf24c0f2d6)

Reply:
We do, otherwise the rootblocks will overflow and the smart highlight button will be hidden at the bottom.